src/BudgetBadgerWebApi.Api/ActionParameters/Expense/CreateExpense.cs

```diff
@@ -0,0 +1,17 @@
+﻿using BudgetBadgerWebApi.Api.ActionParameters.Common;
+using BudgetBadgerWebApi.Application.Logic.Expense.Commands;
+using BudgetBadgerWebApi.Domain.Enums;
+
+namespace BudgetBadgerWebApi.Api.ActionParameters.Expense
+{
+    public record CreateExpense : NamedActionParameter
```

Comment:
I like this approach better, but on the other hand we can have problems later if we want to inheritate from smth else. Another idea is to write this setter logic as our extension method for string. In this way we won't duplicate logic and code will be more readable (at first glance I had wtf where is Name xd) But honestly I'm not sure what is better.